### PR TITLE
Add remove-paywall extension

### PR DESCRIPTION
**Pull request by tegola**
## Description

This adds a simple command to remove the paywall from a URL, either from an argument, from the fallback text or from the selected text. A basic check is in place to make sure the text is a url starting with `http(s)://`.

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/remove-paywall/src/remove-paywall.tsx [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import { Toast, getPreferenceValues, getSelectedText, open, showToast } from "@raycast/api";
+
+interface Preferences {
+  service: string;
+}
+
+export default async () => {
```
Comment (pernielsentikaer) [inline extensions/remove-paywall/src/remove-paywall.tsx:7]: You could also add an argument so the user can write URL directly in Raycast (and maybe also fallbackText) 

Let me know what you think 🙂

Comment (tegola) [reply extensions/remove-paywall/src/remove-paywall.tsx:7]: @pernielsentikaer The bot marked this PR as stale. Could you clarify your requests or just accept the PR as is. I do intend to update the extension in the near future (probably next week).

Comment (tegola) [reply extensions/remove-paywall/src/remove-paywall.tsx]: It seems I can't update this PR, no idea what to do:
```
> npx @raycast/api@latest publish

ready -  validate git repository
ready -  getting fork
ready -  preparing clone
ready -  checking for new contributions
ready -  preparing extension
ready -  pushing extension
error -  submitting extension
open PR: Error:
```

Comment (tegola) [reply extensions/remove-paywall/src/remove-paywall.tsx]: I actually fixed this manually.

**Review by thomaslombart — APPROVED** [review]
🚀

**Pull request merged**